README.md

```diff
@@ -16,9 +16,7 @@
 
 ## Overview
 
-[<img src="https://raw.githubusercontent.com/xdspacelab/openvslam/master/docs/img/teaser.png" width="640px">](https://arxiv.org/abs/1910.01122)
-
-[<img src="https://raw.githubusercontent.com/wiki/xdspacelab/openvslam/media/tracking.gif" width="640px">](https://www.youtube.com/watch?v=Ro_s3Lbx5ms)
```

Comment:
Why not keep this video? Its still online and good context

Reply:
The video's link is still there in the "youtube" logomark. What I erased is a .gif, which was on the wiki page of the original Openvslam repository. In this repository, I believe is set as a "private" page since I can not access or edit.